netContain/BuildContainer.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/bash
+
+function dockerBuildIt {
+    imgId=`docker build $1 | grep "Successfully built" | cut -d " " -f 3`
+
+    if [[ $imgId =~ [0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f] ]]; then
+       echo "$2 Image has been built with ID $imgId"
+       return 0
+    fi
+    echo "$2 Image was not built properly"
+    return 255
+}
+
+set -x 
+
+eexists=`docker images contivbase | grep -w "contivbase" | wc -l`
+if [ $eexists != 0 ]; then
+    echo "An image by name contivbase already exists"
+    echo "Remove contivbase (docker rmi contivbase) and retry"
+    exit
+fi
+
+if [ $1 == "reinit" ]; then
+   etcdctl rm -recursive /contiv.io
+   etcdctl rm -recursive /docker/network
+fi
+
+sudo modprobe openvswitch
```

Comment:
This is regarding modprobe right? 
I found it tricky as just mounting the modules lib does not seem to help, it looks for more things and was throwing errors. 
May help make ubuntu version much more leaner if we get rid of some tools like this. We can modprobe from the build script it self.